mobile/native/src/ln_dlc/mod.rs

```diff
@@ -544,13 +537,23 @@ fn derive_trades_from_filled_orders() -> Result<Vec<WalletHistoryItem>> {
                 .trader_margin()
                 .expect("Filled order to have a margin");
 
+            let execution_price = Decimal::try_from(
+                first
+                    .execution_price()
+                    .context("execution price to be set on a filled order")?,
+            )?;
+            let fee = order_matching_fee_taker(first.quantity, execution_price).to_sat();
+            let amount_sats = amount_sats + fee;
+
             trades.push(WalletHistoryItem {
                 flow,
                 amount_sats,
                 timestamp: first.creation_timestamp.unix_timestamp() as u64,
                 status: Status::Confirmed, // TODO: Support other order/trade statuses
                 wallet_type: WalletHistoryItemType::Trade {
                     order_id: first.id.to_string(),
+                    fee_sat: fee,
```

Comment:
❓Won't it be confusing to display the opening fee and an amount that already includes the opening fee?

Reply:
you mean its confusing that the amount of the transaction does include the fee?